priv/gateway_rs/settings.toml

```
@@ -1,2 +1,8 @@
 ## api listening port override
 api = 4468
+
+[poc]
+entropy_uri = "https://entropy.helium.io/entropy"
+ingest_uri = "http://mainnet-pociot.helium.io:9080"
+interval = 600
+ingest_token = "dev"
```

```suggestion
interval = 600
```
the entropy_uri and ingest_uri are just replicating the defaults so those can be omitted and the lora iot network doesn't use an ingest_token./

the interval can be set as above or if you prefer you could set it in the miner_gateway_port as the environment variable `GW_POC_INTERVAL=600`; either way.

---

Removed the other values